src/pretix/control/forms/orders.py

```diff
@@ -118,3 +118,9 @@ class OrderContactForm(forms.ModelForm):
     class Meta:
         model = Order
         fields = ['email']
+
+
+class OrderLocaleForm(forms.ModelForm):
+    class Meta:
+        model = Order
+        fields = ['locale']
```

Comment:
This results in `locale` being presented as a normal text input, so that I could put down `asdfghklö` as locale. I think it would be preferable to present a `ChoiceField` similar to the one of the [EventWizardBaseForm](https://github.com/pretix/pretix/blob/master/src/pretix/control/forms/event.py#L45).